pkg/bitnet/model/model.go

```diff
@@ -384,10 +248,10 @@ type TransformerBlock struct {
 	FFNNorm  []float32 // FFN normalization weights
 }
 
-// ModelWeights holds all the model's parameters
+// ModelWeights represents all model parameters
 type ModelWeights struct {
 	// Token embeddings (shared with output layer)
-	TokenEmbedding []int8 // Token embedding weights (ternary)
+	TokenEmbeddings []float32
```

Comment:
Is this right? See the project goal at `pkg/bitnet/README.md` and issue #170 . 

We're building BitNet implementation. Should we use floating point numbers?